src/Pages/ResultList/ResultList.js

```diff
@@ -1,7 +1,54 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import ResultListContents from './Component/ResultListContents';
+import SearchBox from './Component/SearchBox';
+import './Resultlist.scss';
 
 const ResultList = () => {
-  return <div>reslis2t</div>;
+  return (
+    <article className="resultListWrap">
```

Comment:
```suggestion
    <article className="resultList">
```
컴포넌트의 가장 최상위 부모태그 클래스네임은 컴포넌트의 이름과 동일하게 설정해주세요!
scss를 사용할 때 위코드 컨벤션입니다!